beacon_node/network/src/network_beacon_processor/mod.rs

```diff
@@ -983,7 +985,7 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                             "result" => "imported block and custody columns",
                             "block_hash" => %hash,
                         );
-                        self.chain.recompute_head_at_current_slot().await;
+                        // Head will be recomputed `handle_lookup_sync_processing_result`
```

Comment:
Feels like it might be easier to move the `recompute_head logic` back here, otherwise we'll have to either do it in two places or having to pass a flag here and conditionally recompute head.